src/parsers/manifest/dash/indexes/helpers.ts

```diff
@@ -110,19 +111,19 @@ function getTimelineRangeEnd({ ts, d, r }: IIndexSegment) : number {
 function getInitSegment(
   index: {
     timescale: number;
-    initialization?: { media?: string; range?: [number, number] };
+    initialization?: { media: string; range?: [number, number] };
     indexRange?: [number, number];
   }
 ) : ISegment {
-  const { initialization = {} } = index;
+  const { initialization } = index;
 
   return {
     id: "init",
     isInit: true,
     time: 0,
-    range: initialization.range || undefined,
+    range: initialization ? initialization.range || undefined : undefined,
     indexRange: index.indexRange || undefined,
-    media: initialization.media,
+    mediaURL: initialization ? initialization.media : "",
```

Comment:
I'm more a fan of ``initialization.media || null``. Here we lie about mediaURL to be defined